The job **`mingw-check-tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/10933978005/job/30353297957) [(plain)](https://github.com/rust-lang/rust/commit/63f14b3a1ef7176e8bef3de4ba21579235cd534a/checks/30353297957/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain

COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/

# NOTE: intentionally uses python2 for x.py so we can test it still works.
# validate-toolstate only runs in our CI, so it's ok for it to only support python3.
ENV SCRIPT TIDY_PRINT_DIFF=1 python2.7 ../x.py test \
           --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
# This file is autogenerated by pip-compile with Python 3.10
# by the following command:
#
#    pip-compile --allow-unsafe --generate-hashes reuse-requirements.in
---
#13 2.844 Building wheels for collected packages: reuse
#13 2.845   Building wheel for reuse (pyproject.toml): started
#13 3.088   Building wheel for reuse (pyproject.toml): finished with status 'done'
#13 3.089   Created wheel for reuse: filename=reuse-4.0.3-cp310-cp310-manylinux_2_35_x86_64.whl size=132715 sha256=dfa09868353292d98f811d3efdb0d54d07389e808efc71d68e3b93c514bf8bec
#13 3.089   Stored in directory: /tmp/pip-ephem-wheel-cache-h5yx731v/wheels/3d/8d/0a/e0fc6aba4494b28a967ab5eaf951c121d9c677958714e34532
#13 3.092 Installing collected packages: boolean-py, binaryornot, tomlkit, reuse, python-debian, markupsafe, license-expression, jinja2, chardet, attrs
#13 3.483 Successfully installed attrs-23.2.0 binaryornot-0.4.4 boolean-py-4.0 chardet-5.2.0 jinja2-3.1.4 license-expression-30.3.0 markupsafe-2.1.5 python-debian-0.1.49 reuse-4.0.3 tomlkit-0.13.0
#13 3.483 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#13 4.016 Collecting virtualenv
#13 4.016 Collecting virtualenv
#13 4.062   Downloading virtualenv-20.26.5-py3-none-any.whl (6.0 MB)
#13 4.262      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 6.0/6.0 MB 30.3 MB/s eta 0:00:00
#13 4.306 Collecting distlib<1,>=0.3.7
#13 4.313   Downloading distlib-0.3.8-py2.py3-none-any.whl (468 kB)
#13 4.323      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 468.9/468.9 KB 56.1 MB/s eta 0:00:00
#13 4.359 Collecting platformdirs<5,>=3.9.1
#13 4.366   Downloading platformdirs-4.3.6-py3-none-any.whl (18 kB)
#13 4.404 Collecting filelock<4,>=3.12.2
#13 4.411   Downloading filelock-3.16.1-py3-none-any.whl (16 kB)
#13 4.492 Installing collected packages: distlib, platformdirs, filelock, virtualenv
#13 4.679 Successfully installed distlib-0.3.8 filelock-3.16.1 platformdirs-4.3.6 virtualenv-20.26.5
#13 DONE 4.8s

#14 [7/8] COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
#14 DONE 0.0s
---
DirectMap4k:      204736 kB
DirectMap2M:    10280960 kB
DirectMap1G:     8388608 kB
##[endgroup]
Executing TIDY_PRINT_DIFF=1 python2.7 ../x.py test            --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
+ TIDY_PRINT_DIFF=1 python2.7 ../x.py test --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
    Finished `dev` profile [unoptimized] target(s) in 0.04s
##[endgroup]
downloading https://ci-artifacts.rust-lang.org/rustc-builds-alt/df7f77811c8806f85522a38878c57fde221138c9/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz
extracting /checkout/obj/build/cache/llvm-df7f77811c8806f85522a38878c57fde221138c9-true/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz to /checkout/obj/build/x86_64-unknown-linux-gnu/ci-llvm
---
   Compiling tidy v0.1.0 (/checkout/src/tools/tidy)
    Finished `release` profile [optimized] target(s) in 28.22s
##[endgroup]
fmt check
Diff in /checkout/library/core/src/mem/maybe_uninit.rs:569:
     /// (Notice that the rules around references to uninitialized data are not finalized yet, but
     /// until they are, it is advisable to avoid them.)
     #[stable(feature = "maybe_uninit", since = "1.36.0")]
-    #[rustc_const_stable(feature = "const_maybe_uninit_as_mut_ptr", since = "CURRENT_RUSTC_VERSION")]
+    #[rustc_const_stable(
+        feature = "const_maybe_uninit_as_mut_ptr",
+        since = "CURRENT_RUSTC_VERSION"
+    )]
     #[cfg_attr(bootstrap, rustc_allow_const_fn_unstable(const_mut_refs))]
     #[inline(always)]
     pub const fn as_mut_ptr(&mut self) -> *mut T {
fmt error: Running `"/checkout/obj/build/x86_64-unknown-linux-gnu/rustfmt/bin/rustfmt" "--config-path" "/checkout" "--edition" "2021" "--unstable-features" "--skip-children" "--check" "/checkout/library/core/src/result.rs" "/checkout/library/core/src/panic.rs" "/checkout/library/core/src/range/iter.rs" "/checkout/library/core/src/range/legacy.rs" "/checkout/library/core/src/ascii.rs" "/checkout/library/core/src/hash/sip.rs" "/checkout/library/core/src/hash/mod.rs" "/checkout/library/core/src/array/iter.rs" "/checkout/library/core/src/array/mod.rs" "/checkout/library/core/src/array/drain.rs" "/checkout/library/core/src/array/ascii.rs" "/checkout/library/core/src/array/equality.rs" "/checkout/library/core/src/convert/num.rs" "/checkout/library/core/src/convert/mod.rs" "/checkout/library/core/src/pin.rs" "/checkout/compiler/rustc_target/src/asm/csky.rs" "/checkout/compiler/rustc_target/src/asm/m68k.rs" "/checkout/compiler/rustc_target/src/asm/hexagon.rs" "/checkout/compiler/rustc_target/src/asm/nvptx.rs" "/checkout/compiler/rustc_target/src/asm/spirv.rs" "/checkout/compiler/rustc_target/src/asm/wasm.rs" "/checkout/compiler/rustc_target/src/asm/msp430.rs" "/checkout/compiler/rustc_target/src/asm/s390x.rs" "/checkout/compiler/rustc_target/src/asm/bpf.rs" "/checkout/compiler/rustc_target/src/asm/x86.rs" "/checkout/compiler/rustc_target/src/asm/arm.rs" "/checkout/compiler/rustc_target/src/asm/avr.rs" "/checkout/compiler/rustc_target/src/asm/mod.rs" "/checkout/compiler/rustc_target/src/asm/loongarch.rs" "/checkout/compiler/rustc_target/src/asm/riscv.rs" "/checkout/compiler/rustc_target/src/asm/powerpc.rs" "/checkout/compiler/rustc_target/src/asm/mips.rs" "/checkout/compiler/rustc_target/src/asm/aarch64.rs" "/checkout/compiler/rustc_target/src/json.rs" "/checkout/compiler/rustc_target/src/tests.rs" "/checkout/library/core/src/intrinsics/mir.rs" "/checkout/library/core/src/intrinsics/simd.rs" "/checkout/library/core/src/mem/maybe_uninit.rs" "/checkout/library/core/src/mem/mod.rs" "/checkout/library/core/src/mem/transmutability.rs" "/checkout/library/core/src/mem/manually_drop.rs" "/checkout/library/core/src/char/decode.rs" "/checkout/library/core/src/char/convert.rs" "/checkout/library/core/src/char/mod.rs" "/checkout/library/core/src/char/methods.rs" "/checkout/library/core/benches/str/char_count.rs" "/checkout/library/core/benches/str/corpora.rs" "/checkout/library/core/benches/str/iter.rs" "/checkout/library/core/benches/str/debug.rs" "/checkout/library/core/benches/net/mod.rs" "/checkout/library/core/benches/net/addr_parser.rs" "/checkout/library/core/benches/tuple.rs" "/checkout/library/core/benches/iter.rs" "/checkout/library/core/benches/slice.rs" "/checkout/library/core/benches/array.rs" "/checkout/library/core/benches/pattern.rs" "/checkout/library/core/benches/any.rs" "/checkout/library/core/benches/lib.rs" "/checkout/library/core/benches/str.rs" "/checkout/library/core/benches/ascii/is_ascii.rs" "/checkout/compiler/rustc_target/src/spec/base/thumb.rs" "/checkout/compiler/rustc_target/src/spec/base/l4re.rs" "/checkout/compiler/rustc_target/src/spec/base/linux_gnu.rs" "/checkout/library/core/src/bool.rs"` failed.
If you're running `tidy`, try again with `--bless`. Or, if you just want to format code, run `./x.py fmt` instead.
  local time: Thu Sep 19 03:40:24 UTC 2024
  network time: Thu, 19 Sep 2024 03:40:24 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/library/core/src/mem/maybe_uninit.rs
+++ b/library/core/src/mem/maybe_uninit.rs
@@ -569,7 +569,10 @@ impl<T> MaybeUninit<T> {
     /// (Notice that the rules around references to uninitialized data are not finalized yet, but
     /// until they are, it is advisable to avoid them.)
     #[stable(feature = "maybe_uninit", since = "1.36.0")]
-    #[rustc_const_stable(feature = "const_maybe_uninit_as_mut_ptr", since = "CURRENT_RUSTC_VERSION")]
+    #[rustc_const_stable(
+        feature = "const_maybe_uninit_as_mut_ptr",
+        since = "CURRENT_RUSTC_VERSION"
+    )]
     #[cfg_attr(bootstrap, rustc_allow_const_fn_unstable(const_mut_refs))]
     #[inline(always)]
     pub const fn as_mut_ptr(&mut self) -> *mut T {

